### DISCUSSION TITLE
Skyscraper Wallpaper

### DISCUSSION THREAD

**Discussion by uncognic** [Atlas-OS/Atlas#114] [Community Artwork]
heres a wallpaper i made

![Atlas Landscape](https://user-images.githubusercontent.com/92607100/156785637-2bca7b26-08b6-4f6c-a9c9-5d497c424b89.jpg)

**Comment by Dust52-arch** [comment]
Как скачать?

**Comment by uncognic** [comment]
нажмите на изображение, затем щелкните правой кнопкой мыши и скачайте

-------- Original Message --------

**Comment by Dust52-arch** [comment]
а ну да Спасибо)

ср, 15 янв. 2025 г. в 02:25, uncognic ***@***.***>:
